backend/src/main/java/hu/flowacademy/meteo/bootstrap/InitDataLoader.java

```diff
@@ -22,182 +21,109 @@
 @RequiredArgsConstructor
 public class InitDataLoader implements CommandLineRunner {
 
-    private final AirDataRepository airDataRepository;
-    private final BatteryDataRepository batteryDataRepository;
-    private final MiscDataRepository miscDataRepository;
-    private final SoilDataRepository soilDataRepository;
-    private final WindDataRepository windDataRepository;
     private final StationRepository stationRepository;
-    private final MeasurmentRepository measurmentRepository;
+    private final MeasurementRepository measurementRepository;
 
-    private static final DateFormat DATE_FORMAT_HU = new SimpleDateFormat("yyyy.MM.dd HH:mm", Locale.forLanguageTag("HU-hu"));
-    private static final DateFormat DATE_FORMAT_HU_SPACED = new SimpleDateFormat("yyyy. MM. dd. HH:mm", Locale.forLanguageTag("HU-hu"));
-    private static final String[] FILE_NAME = {"CSIHA_HQ_10perc.csv", "CSIHA_HQ_orai.csv", "CSIHA_HQ_napi.csv", "public_allomasok.csv"};
-    private static final String HOME_STATION_NAME = "Szeged";
+    private static final String DATE_FORMAT_HU = "yyyy.MM.dd HH:mm";
+    private static final String DATE_FORMAT_HU_SPACED = "yyyy. MM. dd. HH:mm";
+    private static final String DATE_FORMAT_HU_DASH = "yyyy-MM-dd HH:mm";
+    private static final String[] FILE_NAME_TEN = {"CSIHA_HQ_10perc.csv", "D_Baja_10perc.csv", "D_CSANÁDPALOTA_10perc.csv"};
```

Comment:
Ha itt nem csináljátok meg, akkor vegyetek fel pls egy storyt arra, hogy ne a jar-ba csomagoljuk be az adatokat (ahogyan most történik). Ezek input file-ok, simán lehetne a Dockerfile-ban a rendszer alá egy volume-ba betenni őket és onnan felolvasni, ez így nem korrekt. Másik megoldás - ami szebb is lenne - hogy egy endpointon keresztül töltitek be ezeket az adatokat. Jelenleg adatbővülés esetnén nem csak deployolni kell (ami már magában gáz), hanem még a kódot is átírni. Ez így egyáltalán nem professzionális.

Reply:
Ha vesztek fel külön story-t, akkor ettől függetlenül menjen be.

Reply:
Igen, egy adatfeltöltés (user által) a jövő héti feladat, melyhez természetesen kell majd endpoint is ami csv-t vár. Most a 'használhatóság' miatt kerül még be pár állomásról származó adat.

Reply:
Ha így részedről rendben @akesztyus, akkor részemről is :)